pkg/metrics/metrics.go

```diff
@@ -787,6 +791,16 @@ func CreateConfiguration(metricsEnabled []string) (Configuration, []prometheus.C
 			collectors = append(collectors, EventTSK8s)
 			c.EventTSK8sEnabled = true
 
+			EventLagK8s = prometheus.NewGauge(prometheus.GaugeOpts{
+				Namespace:   Namespace,
+				Name:        "event_lag_seconds",
+				Help:        "Lag (computed value) for Kubernetes events",
+				ConstLabels: prometheus.Labels{"source": LabelEventSourceK8s},
+			})
```

Comment:
Have you run this through `promtool`?

Reply:
Also, I think we can add more context to the description of this metrics. Since this is the lag between receiving the CNI ADD and then getting it in the cache, let's add that so it is clear for readers.

Reply:
I haven't but our smoke tests do :)